lib/utils/hasPrefix.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const { prefixes } = require('../reference/prefixes');
+
+const HAS_PREFIX_REGEX = new RegExp(`(?:${[...prefixes.values()].join('|')})`, 'i');
```

Comment:
[suggest] Sorry, I found it after approve. 😅 We can omit `.values()`:

```suggestion
const HAS_PREFIX_REGEX = new RegExp(`(?:${[...prefixes].join('|')})`, 'i');
```